### ISSUE TITLE
Updated  (JDT language server 1.2.0-202106301459 #337) but still getting error with jdtls

### ISSUE THREAD

**Issue by paradigma0621** [emacs-lsp/lsp-java#339]
**Describe the bug**
An oldest version of ldtls works fine with typical java projects. Updating it to lasted version, or with jdtls 1.2.0-202106301459 it gives the error for debugging the java projects:

Warning (emacs): Initialize request failed: 'void org.eclipse.jdi.internal.connect.ConnectorImpl$StringArgumentImpl.<init>(org.eclipse.jdi.internal.connect.ConnectorImpl, java.lang.String, java.lang.String, java.lang.String, boolean)'

  **Note**
When just changing the new /home/lucas/.emacs.d/.cache/lsp/eclipse.jdt.ls to the older one it works.

**Comment by huangfeiyu** [comment]
@paradigma0621 I know there are two things may related to the issue
1. the jdtls requires java 11 to run after version 0.57, 0.57 is the last version runs in java 8.
2. The jdtls version you use has problem with lsp-java, jdt-language-server-1.1.0-202104291503 works fine with the latest lsp-java, it is the version I have used for about a month without major problem.
3. remove the directory ~/.emacs.d/workspace/ after you setup new jdtls
Also please refer the following article to setup lsp-java if you want to run jdtls on java 11 but use java 8 for your project. hope these will help you.

**Comment by paradigma0621** [comment]
Running with... Java 15
For the same Java installation it runs in the old jdtls, but not in the newer one.

**Comment by huangfeiyu** [comment]
@paradigma0621 yes, I encountered issue with jdtls 1.2.0-202106301459, after I replace it with jdtls 1.1.0-202104291503, it works quit well.
